backend/index.js

```
@@ -1,3 +1,9 @@
+require('dotenv').config();
+const chatbotRoutes = require('./routes/chatbot');
+app.use('/api/chat', chatbotRoutes);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)